ipatests/prci_definitions/temp_commit.yaml

```diff
@@ -47,28 +47,28 @@ topologies:
     memory: 14500
 
 jobs:
-  fedora-latest-ipa-4-8/build:
+  fedora-latest/build:
     requires: []
     priority: 100
     job:
       class: Build
       args:
         git_repo: '{git_repo}'
         git_refspec: '{git_refspec}'
-        template: &ci-ipa-4-8-latest
-          name: freeipa/ci-ipa-4-8-f32
-          version: 0.0.9
+        template: &ci-master-latest
```

Comment:
If the PR is made against ipa-4-8 branch, please use the ci-ipa-4-8-latest template to launch the tests.